core/src/main/java/com/linecorp/armeria/client/endpoint/EndpointGroup.java

```diff
@@ -52,7 +54,7 @@ static EndpointGroup empty() {
      * Waits until the initial {@link Endpoint}s are ready.
      *
      * @throws java.util.concurrent.CancellationException if {@link #close()} was called before the initial
-     * {@link Endpoint}s are set
+     *     {@link Endpoint}s are set
```

Comment:
remove white space?

Reply:
@minwoox  asked to indent :) Admittedly I'm not sure what the indentation rules of javadoc are

Reply:
```
@param myParam description first line
               second line
@throws FooException description first line
                     second line
```

..?

Reply:
Sorry about the comment that I left. It was unclear.
I think we need to align the `{` under the `if`. :)